api/api.go

```diff
@@ -360,6 +360,14 @@ func (api *Server) SendAction(ctx context.Context, in *iotexapi.SendActionReques
 	if err = selp.LoadProto(in.Action); err != nil {
 		return nil, status.Error(codes.InvalidArgument, err.Error())
 	}
+
+	// reject action if chainID is not matched at JutlandHeight
+	if api.cfg.Genesis.Blockchain.IsJutland(api.bc.TipHeight()) {
```

Comment:
after https://github.com/iotexproject/iotex-core/pull/2782 merge, change to `IsKamchatka()`

Reply:
#2782 has merged, change to IsKamchatka()